### PR TITLE
fix: android q landscape height

### PR DESCRIPTION
**Pull request by markhomoki**

### CODE REVIEW

File: lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/ComponentViewController.java [hunk 1]
```diff
@@ -114,7 +116,11 @@ public void applyBottomInset() {
 
     @Override
     public int getBottomInset() {
-        int navigationBarInset = resolveCurrentOptions().statusBar.isHiddenOrDrawBehind() ? 0 : NavigationBarUtils.getNavigationBarHeight(getActivity());
+        Context activity = getActivity();
+        boolean isPortrait = activity.getResources().getConfiguration().orientation == Configuration.ORIENTATION_PORTRAIT;
+        boolean shouldSetBottomInset = !(resolveCurrentOptions().bottomTabsOptions.isHiddenOrDrawBehind() || resolveCurrentOptions().statusBar.isHiddenOrDrawBehind());
+        int navigationHeight = NavigationBarUtils.getNavigationBarHeight(activity);
```
Comment (ahuounan) [inline lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/ComponentViewController.java]:
```suggestion
        int navigationBarHeight = NavigationBarUtils.getNavigationBarHeight(activity);
```

File: lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/ComponentViewController.java [hunk 2]
```diff
@@ -114,7 +116,11 @@ public void applyBottomInset() {
...
+        int navigationBarInset = !shouldSetBottomInset && isPortrait ? 0 : navigationHeight;
```
Comment (ahuounan) [inline lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/ComponentViewController.java]: tested on android tablet w/ gesture navigation?

**Review by marceltex — APPROVED** [review]

File: lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/ComponentViewController.java (hunk 2 above)
Comment (ndillon1) [reply lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/ComponentViewController.java]: This is one of those changes that we need to test across lots of different android devices

**Review by ahuounan — APPROVED** [review]
what Marcel said

**Review by ndillon1 — APPROVED** [review]

**Pull request merged**